### PR TITLE
Boeing viz/hover function

### PR DESCRIPTION
**Pull request by dlousa92**
Hey Jacque, This is the hover effect we talked about adding. I also adjusted how the scroll text looks (changing background color to gray, etc). I added a bit more margin between the paragraphs allowing the user a moment to look at all the associated pieces with no paragraph in the way. 

Finally I adjusted the 'viewed' opacity a bit so users can see more clearly which parts they're currently looking at.

### CODE REVIEW

File: assets/_sass/custom-viz/boeing/_base.scss [hunk 1]
```diff
@@ -82,10 +88,11 @@ $size_boeing-container-max-width: 700px;
...
     .part {
       margin: 0.1rem;
-      background-color: $yellow;
+      border-bottom: 1px solid $yellow;
 
       &:hover {
-        background-color: $dark-yellow;
+        color: $dark-gray;
+        background-color: $yellow;
         transition: all 0.2s ease-in-out;
```
Comment (jnschrag) [inline assets/_sass/custom-viz/boeing/_base.scss]: Remember that transitions need to be applied to the base class, not the `:hover` state.

**Comment by dlousa92** [comment]
Thanks for those suggestions! Great idea with refactoring how the highlight function is called. Removes unnecessary doubled up code

File: assets/_sass/custom-viz/boeing/_scrollytelling.scss [hunk 2]
```diff
@@ -5,14 +5,14 @@
 
   .svgPart {
     opacity: 0.2;
-    transition: opacity 0.7 linear;
+    transition: opacity 0.7s linear;
```
Comment (jnschrag) [inline assets/_sass/custom-viz/boeing/_scrollytelling.scss]: Can we speed this up to .4s? Same with the transitions on `.svgLabel`? .7s is a little bit too long.

Comment (dlousa92) [reply assets/_sass/custom-viz/boeing/_scrollytelling.scss]: Yep!

File: assets/_sass/custom-viz/boeing/_scrollytelling.scss [hunk 3]
```diff
@@ -19,7 +19,7 @@
   .svgLabel {
     visibility: hidden;
     opacity: 0;
-    transition: visibility 0.7s linear, opacity 0.7s linear;
+    transition: visibility 0.4s linear, opacity 0.7s linear;
```
Comment (jnschrag) [inline assets/_sass/custom-viz/boeing/_scrollytelling.scss]: Don't forget to change the `opacity` duration value too!

Comment (dlousa92) [reply assets/_sass/custom-viz/boeing/_scrollytelling.scss]: whoops!

**Pull request merged**